### DISCUSSION TITLE
Possibility to not expose the docs on specific subdomains?

### DISCUSSION THREAD

**Discussion by eroctra** [dedoc/scramble#115] [Q&A] [answered]
My Laravel application has 3 subdomains: api, cms and app. 

Right now the api docs are exposed for every subdomain. 
Ideally I would like to only expose the docs for the api subdomain. 

Is there any way to archive this?

**Comment by romalytvynenko** [comment, marked as answer]
@eroctra hey. Sorry for the late reply

Currently there is no way to do this, but I've created an issue to make it possible: https://github.com/dedoc/scramble/issues/143

If still needed, please track the issue status.

🙌
